src/commands/init.js

```diff
@@ -146,6 +147,66 @@ async function init() {
 const sleep = (time: number = 1000) =>
   new Promise(resolve => setTimeout(resolve, time));
 
+const getRunScript = (scriptName: string) => {
+  const runCommand = scriptName === 'start' ? 'yarn' : 'yarn run';
+  return `${runCommand} ${scriptName}`;
+};
+
+/**
+ * Adds Haul to package.json scripts
+ */
+const addToPackageScripts = async (cwd: string) => {
+  const pjson = JSON.parse(
+    fs.readFileSync(path.join(cwd, 'package.json')).toString(),
+  );
+
+  const scripts = pjson.scripts || {};
+
+  const haulScript = Object.keys(scripts).find(name =>
+    scripts[name].includes('haul'));
```

Comment:
This will match words with the text `haul` in them. Probably better just to check `scripts[name] === 'haul start'`